reporters/uniswapSubgraphClient.js

```diff
@@ -0,0 +1,58 @@
+// TODO: We should group together graphql related modules but I'm using this location
+// for now because this is the only use case.
+
+const { GraphQLClient } = require("graphql-request");
+
+/**
+ * Return created graphql client or create a new one.
+ * @return `createdClient` singleton graphql client.
+ */
+let uniswapClient;
+const getUniswapClient = () => {
+  if (!uniswapClient) {
+    uniswapClient = new GraphQLClient("https://api.thegraph.com/subgraphs/name/ianlapham/unsiwap3");
```

Comment:
Just curious -- are graph apis immutable? Put another way, is this client always guaranteed to have the same data or can the schema be changed?

Reply:
I don't know but I would not assume its immutable. I would say that we should catch errors here, but we don't catch errors throughout the DailyReporter script except at the root level (`index.js: run()`)

Reply:
SGTM -- just curious.